test/browser.js

```diff
@@ -0,0 +1,28 @@
+import path from 'path';
+import fs from 'fs';
+import test from 'ava';
+import execa from 'execa';
+import objectAssign from 'object-assign';
```

Comment:
Hmm. I thought this doesn't work because `transform-runtime` causes Babel not to polyfill. I think you're right though. This is a static method. Transform runtime should work.